lib/unicore/mktables

```diff
@@ -10088,7 +9743,7 @@ sub Standardize($) {
     return $name;
 }
 
-sub standardize ($) {
+sub standardize ($str) {
```

Comment:
You left the shift in in this sub

Reply:
indeed and need to set the default to undef, as the function return undef on undef input.. 
fixing it